eZ/Publish/API/Repository/Tests/ContentTypeServiceTest.php

```diff
@@ -4367,4 +4367,24 @@ public function testUpdateContentTypeDraftWithNewTranslationWithMultilingualData
             $loadedFieldDefinition->fieldSettings['multilingualOptions']
         );
     }
+
+    /**
+     * Test for the deleteUserDrafts() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentTypeService::deleteUserDrafts()
+     * @expectedException \eZ\Publish\API\Repository\Exceptions\NotFoundException
```

Comment:
`$this->expectException('NotFoundException');` I think we used to do it this way

Reply:
not in this file , in this file it is done by annotations

Reply:
Maybe this is not about which file it is but which is better, more bulletproof or recommended https://thephp.cc/news/2016/02/questioning-phpunit-best-practices

Reply:
For example, in `UserServiceTest`, `LanguageServiceTest` and others we have mixed, and everything is alright

Reply:
This is about one convention in the project. if we want to change convention it's fine with me. But not in this PR and for all occurrences. @alongosz

Reply:
>> `$this->expectException('NotFoundException');` I think we used to do it this way
>
> not in this file , in this file it is done by annotations

@pawbuj @mikadamczyk is right. It's pointless to use `@expectException`. It's been deprecated in further PHPUnit versions and either has been dropped already or will be. So right now new code using that is technical debt.

Reply:
Yes, for some time now we have such a convention to use $this->expectException in new PR-s, right @alongosz ?

Reply:
this is most recently merged PR that contains expectedException https://github.com/ezsystems/ezpublish-kernel/pull/2705

Reply:
And what's the point of bringing this up with the comment that is there?
https://github.com/ezsystems/ezpublish-kernel/pull/2705/files#r305391089

Reply:
the point is: 2 days ago code that contains @xpectedException was merged int 7.5

Reply:
also point is : both methods are acceptable , but annotation follows file convention

Reply:
@pawbuj, ok, but please remember to change this on merge up. For me it would be easier to change this here. It's already a technical debt.

Reply:
Sorry but there is no point in using annotations. It will log warnings in newer PHPUnit. I can easily see how it will slip into master. You can easily forget to change it, especially when we do not make PR when merging up.